txnpool/proc/txnpool_worker.go

```diff
@@ -176,23 +164,15 @@ func (worker *txPoolWorker) verifyTx(tx *tx.Transaction) {
 	}
 
 	if _, ok := worker.pendingTxList[tx.Hash()]; ok {
-		log.Debugf("verifyTx: transaction %x already in the verifying process",
-			tx.Hash())
+		log.Debugf("verifyTx: transaction %x already in the verifying process", tx.Hash())
 		return
 	}
-	// Construct the request and send it to each validator server to verify
-	req := &types.CheckTx{
-		Tx: tx,
-	}
 
-	worker.sendReq2Validator(req)
+	worker.startFullVerify(tx)
```

Comment:
I think safe order is put to map first and then start verify, in case when verify process is so fast, the comming back response will not found in pendingTxList